testing/iframe.js

```diff
@@ -152,7 +152,7 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
       };
       let timeout = setTimeout(function() {
         reject(new Error('Timeout waiting for elements to start loading.'));
-      }, 2000);
+      }, process.env.TEST_TIMEOUT);
```

Comment:
I don't know if `process` is propagated to the browser by Karma. We do a few things like this elsewhere, but I can't find it on my phone.